src/nodes/HardBreak.ts

```diff
@@ -0,0 +1,43 @@
+import Node from "./Node";
+
+export default class HardBreak extends Node {
+  get name() {
+    return "br";
+  }
+
+  get schema() {
+    return {
+      inline: true,
+      group: "inline",
+      selectable: false,
+      parseDOM: [{ tag: "br" }],
+      toDOM() {
+        return ["br"];
+      },
+    };
+  }
+
+  commands({ type }) {
+    return () => (state, dispatch) => {
+      dispatch(state.tr.replaceSelectionWith(type.create()).scrollIntoView());
+      return true;
+    };
+  }
+
+  keys({ type }) {
+    return {
+      "Shift-Enter": (state, dispatch) => {
```

Comment:
We should be able to use `Cmd` + `Enter` here, the trick is to check if we're inside a table first and only handle the event then – search the codebase for `isInTable` helper.

This will also fix an issue this has created where if you add a hard break in a regular paragraph tag it's broken when you reload.

Reply:
I can't use Cmd+Enter because it's already used to save the document.
I chose Shift+Enter because it's a common shortcut to new lines in order editors.

Reply:
Yea, you're totally right.